### ISSUE TITLE
Yeelight always retains old scene when applying new scene

### ISSUE THREAD

**Issue by t-anjan** [diyhue/diyHue#151]
**Describe the bug**
I have many Yeelight bulbs. They always turn on to the same scene as they were when switched off, even if we apply a different scene when turning them on.

**To Reproduce**
Suppose I apply a scene (say `Scene 1`) on a room containing these bulbs, and later, suppose I turn off the bulbs. Next time:

* If I just "turn on" the lights (not applying a scene), they always turn on to the same brightness and colour as they were when I had switched them off (`Scene 1`). This is fine.
* If I "apply a scene" (`Scene 2`, different scene from when the bulbs were turned off) without turning on the bulbs first, the bulbs still turn on to `Scene 1`. Another tap on `Scene 2` button makes the bulbs switch over to `Scene 2`.

**Expected behavior**
When the bulbs are off, tapping on any scene button should apply that scene correctly on the bulbs, immediately. It should not need a second tap.

**Additional context**
Please let me know if you need any additional information. This is extremely annoying when used in conjunction with a motion sensor. After no movement is detected for the set time interval, the lights get dimmed for around 15 seconds and then turn off. Next time there is motion, the bulbs turn on in the "dimmed" mode.

**Comment by mariusmotea** [comment]
Can you perform following test?

in file protocols/yeelight.py line 113 and 114 you have following code:

```
    for key, value in payload.items():
        command(address["ip"], key, value)
```

append a sleep after command line:

```
    for key, value in payload.items():
        command(address["ip"], key, value)
        sleep(0.5)
```

play with this value until you obtain the minimum delay possible. I thing the issue is that the commands are send too fast. Is not possible to set on the color temperature + brightness in the same time. For color scenes i don't believe it is a problem because rgb data is sent in one shoot to the bulb and this contain also the brightness.

**Comment by t-anjan** [comment]
I tried your change but it didn't work. With the sleep command added to the code, the temperature (colour) of the scene is not applied at all. 

I did a little more testing without the sleep command. I just figured out the following pattern in the behaviour:

* Suppose I set the "Dimmed" (default Hue) scene. It shows a very dim **yellow** colour on the bulb, which is correct. Then I turn off the bulb.
* Then, without turning on the bulb, suppose I set the "Energize" scene. It should make the bulb bright white. But it shows a very dim **white** colour. 
* If I tap on the Energize scene button again, the bulb changes to bright white.

In other words, the colour of the scene is set correctly (changed from yellow to white). Just the brightness of the scene is not set.

This behaviour is consistent with all scene combinations.

**Comment by mariusmotea** [comment]
yes, is exactly what i expect. Just the first and probably the second commands are applied, last one is ignored. Still there is no reason to have it not working because of extra sleep command added, probably you had indent error and the entire function was not executed. Please try again with sleep there. Put sleep(1) at the beginning that will add 1 second delay between the commands.
